### ISSUE TITLE
Ideas for OpenCon Receptions

### ISSUE THREAD

**Issue by JosephMcArthur** [sparcopen/opencon#90]
Our rough objectives for receptions historically have been to:
* Make the most of the time attendees have together
* Help people network and get to know each other more informally
* Make sure people get fed, have time to relax, and get enough sleep!

In previous years OpenCon has leaned on receptions based around alcohol for our evenings. For a number of reasons, we're interested in new ideas for how to do receptions this year. Would be great to hear from people on:

* If you have ideas for things we could try
* If you've seen other styles of receptions "work", or (most importantly) not work!

I should note, we're not necessarily thinking we won't do any of our usual receptions this year, but we're interested in adding variety. We're in parallel going to make sure our more traditional receptions better foster a safe, inclusive environment by design. 

Tagging @sparcopen/staff-opencon & @sparcopen/oc-opencon for thoughts. 

To get the ball rolling, here are a few things I thought while writing this issue:
* idea: City tours (or tours of related places, for example, I once was at an event where they took us to a maker space)
* idea: Evening doathon or more informal programming
* idea: A board game night
* idea: Say we'll have spaces, and let attendee's book it for whatever they'd like to do. Or, generally just ask the people attending. 
* Something where we try and get folks to be a bit creative and silly (maybe, some arty activity or something like that).
* Are there evening activities that are better for introverts (not at all suggesting this out of self-interest...)
* Personally I prefer receptions that don't need people to bring things (like costumes, food etc) or plan in any way
* It's likely that whatever we do, some people will want to go to a bar and may do that anyway.

**Comment by rchampieux** [comment]
I like the idea of low key activities that get people talking and getting silly - something like board game night would be fun, especially if there were a variety to choose from that would appeal different styles.  

City tours are fun, but maybe not on the first night - I'm not sure it would get people talking to each other.  

Some additional networking/activity ideas for receptions:

* DIY sign making for the conference
* Decorate your badge station
* Slide deck Karaoke
* Get to know each other bingo

**Comment by npscience** [comment]
+1 to decorate your badge. Alternative: design your own laptop sticker?

City tour/walk - sounds good to me, I've seen this done on first night before. I wonder if you could expand on your concern @rchampieux? Is it because it needs to happen in smaller groups?

I think a facilitated activity (rather than an open room for networking/reception) could be good for first night as it could be more structured / less intimidating. Negative could be having a particular start time (different arrival times).

+1 to sober-friendly activities - maybe to sit as much with as alongside ones that allow people to drink if they want to?

**Comment by rchampieux** [comment]
@npscience re city tours - because the focus the focus would be so outward, I think people would have less opportunities for striking up sustained conversations with people they don't already know.  Maybe I'm overthinking it 😸 , especially if you've seen it done well on the first night.  

+1 to design your own laptop sticker!

**Comment by rchampieux** [comment]
One more - very specific idea.  I've done a couple meet and greet events where we set up [Makey Makey](http://makeymakey.com/) stations and people worked together to create silly applications, like a banana piano or super simple video games.  They have fun and get know each other by putting stuff together, but also playing with what's been created.

**Comment by char-siuu-bao** [comment]
I kind of like city tours (or food tours!) I remember in OpenCon 2014, we had busses shuttle people to different restaurants. People could order drinks with dinner if they wanted to, but it definitely wasn't boozy or centred around alcohol (plenty of people didn't drink), and it was good way to mingle in an informal setting (although maybe still a little anxiety inducing for introverts).  Could that be worth exploring again?

**Comment by thomasmboa** [comment]
Hi everyone,

All these activities sound good for me, but I can add +1 to  Evening
doathon and propose these activity:  *How to use rpi3
<https://www.raspberrypi.org/> to advance OOO? *

I don't know if there are soccer fans in the group, but Germany is one of
the greatest country of soccer in world, can we propose something related
to it?

😃😃😃


2017-08-31 14:07 GMT-04:00 Lorraine Chuen <notifications@github.com>:

**Comment by rchampieux** [comment]
+1 to soccer @thomasmboa 
I really like the idea of food tours too @lorrainechu3n.

**Comment by pennybphd** [comment]
Berlin Food Stories do food tours, which I've always fancied (they have set
dates but also do personal/group tours). I quite like the idea of there
being a range of things to choose from, led by OC etc members, to suit
different needs and interests. Sport, board games, tech stuff, city tour,
food tour, music, quiz, karaoke...

I've always wanted to do this, of course:
http://www.musictours-berlin.de/en/bowie-berlin-walk

**Comment by JosephMcArthur** [comment]
To massively steal an idea from another thread, I think an evening would be a great time to make a MASSIVE zine (as suggested in #80). Folks can get creative, in a chilled way, while keeping a bit of a focus on open & what not. Also, I think it achieves the networking thing in a not awful way - "hey, what are you drawing".

**Comment by npscience** [comment]
+1 to make-a-zine!!

+1 also to the range of activities suggested above (Penny, I'd Bowie walk with you!!) - could we have an "activity evening" and OC alum (and confident newcomers) can propose things (and bring/arrange required props)?

We'd need a good park near the hotel for football, and maybe a room in the hotel or nearby for any inside activity.

**Comment by aldirdiri** [comment]
I really like the Food Tour idea (like the one we had in 2014). It did some good job in networking. I also like the other activities mentioned above (especially the sports one or maybe a Karaoke). Making a zing could be also useful in getting some work done.

The city tour might not be really effective. I attended a conference in DC last year were had a city tour but people ended up sticking to groups of 2-3.

**Comment by JosephMcArthur** [comment]
I recently dug this out from my may conference trinkets. Might be an idea that's helpful to encourage networking/conversation without booze. 

![img_20170917_171617311](https://user-images.githubusercontent.com/4953703/30522685-1f593d3a-9bcc-11e7-954f-538488475f58.jpg)
![img_20170917_171627068](https://user-images.githubusercontent.com/4953703/30522686-1f6fac6e-9bcc-11e7-9802-9036f8defe0d.jpg)

**Comment by nshockey** [comment]
Bumping this thread up @sparcopen/staff-opencon. Any ideas for this that we should use on the Saturday evening reception? Or for the closing reception on Monday? Both will be at the Harnack House immediately following the end of the day.

**Comment by RaoOfPhysics** [comment]
Been out of the loop, but do we still need ideas for this? If so, how formal/informa/semi-formal are we talking?

**Comment by nshockey** [comment]
Right now the plan is to have them be informal, but still open to ideas if folks have 'em!

**Comment by JosephMcArthur** [comment]
Thanks all, no more time for input here!
